numpy/ma/core.py

```diff
@@ -786,22 +786,17 @@ def fix_invalid(a, mask=nomask, copy=True, fill_value=None):
 
 class _DomainCheckInterval(object):
     """
-    Define a valid interval, so that :
-
-    ``domain_check_interval(a,b)(x) == True`` where
-    ``x < a`` or ``x > b``.
+    Define a valid interval, so that:
 
+    ``_DomainCheckInterval(a,b)(x)`` is False where ``a <= x <= b``.
```

Comment:
Also, while we're updating the comment, I'm not sure I like "define a valid interval". maybe "define an interval of validity"?